src/OneDrive.Sdk.Authentication.Desktop/Business/AdalDaemonAuthenticationProvider.cs

```diff
@@ -44,8 +44,8 @@ public class AdalDaemonAuthenticationProvider : AdalAuthenticationProviderBase
             _clientKey = clientSecret;
 
             string authority = String.Format(CultureInfo.InvariantCulture, "https://login.microsoftonline.com/{0}", tenant);
-            authContextWrapper = authenticationContextWrapper;
-            clientCredential = new ClientCredential(_clientId, _clientKey);
+            this.authContextWrapper = authenticationContextWrapper;
+            this.clientCredential = new ClientCredential(_clientId, _clientKey);
```

Comment:
`this.clientCredential = new ClientCredential(this._clientId, this._clientKey);`

Please look for this everywhere in the file. Instance members and methods should always have `this.`